apps/shipments/signals.py

```diff
@@ -66,14 +81,18 @@ def shipment_post_save(sender, **kwargs):
         signature = rpc_client.add_shipment_data(instance.storage_credentials_id, instance.shipper_wallet_id,
                                                  instance.vault_id, ShipmentVaultSerializer(instance).data)
 
+        LOG.debug(f'Updating LOAD contract with vault uri/hash {signature["hash"]}.')
         # Update LOAD contract with vault uri/hash
         instance.update_vault_hash(signature['hash'])
 
 
 @receiver(post_save, sender=LoadShipment, dispatch_uid='loadshipment_post_save')
 def loadshipment_post_save(sender, **kwargs):
     instance, created = kwargs["instance"], kwargs["created"]
+    LOG.debug(f'LoadShipment post save with sender {sender}.')
```

Comment:
The `sender` of this signal is just a `LoadShipment` class so this log isn't really informative. Maybe log `instance.id` (LoadShipment.id) instead